File: week3/lab_java/Exercise4.java
package lab3;

import java.util.Scanner;

public class Exercise4 {
    static Scanner in = new Scanner(System.in);

    public static void main(String[] args) {
        System.out.println("Please enter the x1 coordinate of point A: ");
        float x1 = in.nextFloat();
        System.out.println("Please enter the y1 coordinate of point A: ");
        float y1 = in.nextFloat();
        System.out.println("Please enter the x2 coordinate of point B: ");
        float x2 = in.nextFloat();
        System.out.println("Please enter the y2 coordinate of Point B: ");
        float y2 = in.nextFloat();

        System.out.printf("The distance between A(%.2f,%.2f) and B(%.2f,%.2f) is %.2f.\n", x1,y1,x2,y2,Math.sqrt(Math.pow(x2-x1,2) + Math.pow(y2-y1,2)));
    }
}

File: week3/lab_py/exercise4.py
import math as m 

x1 = float(input("x1 = "))
y1 = float(input("y1 = "))
x2 = float(input("x2 = "))
y2 = float(input("y2 = "))
distance = m.sqrt(pow(x1-x2,2) + pow(y1-y2,2))
print(f'The distance between A({x1},{y1}) and B({x2},{y2}) is: {distance:.2f}')